### PR TITLE
Migration guide for SDK v5.0.0

### PR DESCRIPTION
**Pull request by sergiishamrai-okta**
## Category

- [x] Documentation

SDK supports openapi-spec v2.6.0
New Interfaces and Enums has been added.

### CODE REVIEW

File: MIGRATING.md [hunk 1]
```diff
@@ -2,6 +2,212 @@
  
 This SDK uses semantic versioning and follows Okta's [library version policy](https://developer.okta.com/code/library-versions/). In short, we do not make breaking changes unless the major version changes!
 
+## Migrating from 4.x.x to 5.0.0
```
Comment (bdemers) [inline MIGRATING.md:5]: 5.0 should be 4.1 ?

**Review by bdemers — CHANGES_REQUESTED** [review]

**Comment by bdemers** [comment]
IMHO, this depends on #590 getting re-opened and merged first
cc: @arvindkrishnakumar-okta

**Comment by arvindkrishnakumar-okta** [comment]
> IMHO, this depends on #590 getting re-opened and merged first
> cc: @arvindkrishnakumar-okta

Agreed.

**Review by arvindkrishnakumar-okta — CHANGES_REQUESTED** [review]
See open [comment](https://github.com/okta/okta-sdk-java/pull/589#issuecomment-845190386) above.

File: MIGRATING.md [hunk 2]
```diff
@@ -2,6 +2,212 @@
  
 This SDK uses semantic versioning and follows Okta's [library version policy](https://developer.okta.com/code/library-versions/). In short, we do not make breaking changes unless the major version changes!
 
+## Migrating from 4.x.x to 4.1.x
+
+Version 5.0.0 of this SDK introduces a number of breaking changes from previous versions.
```
Comment (arvindkrishnakumar-okta) [inline MIGRATING.md:7]:
```suggestion
Version 4.1.x of this SDK introduces a number of breaking changes from previous versions.
```

**Review by arvindkrishnakumar-okta — CHANGES_REQUESTED** [review]
see comments.

**Review by arvindkrishnakumar-okta — APPROVED** [review]

**Pull request merged**